spec/factories/tulsa/inmates.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :inmate do
+    first { Faker::Lorem.word }
+    middle { Faker::Lorem.word }
+    last { Faker::Lorem.word }
+    gender { Faker::Lorem.word }
+    roster { '' }
```

Comment:
if you leave this off completely I think it auto generates the assocation
(so just remove the {  '' }